CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # MicroProfile Language Server Changelog
 
+## [0.5.0](https://github.com/eclipse/lsp4mp/milestone/5?closed=1) (July 28, 2022)
+### Enhancements
+ * performance - Delay revalidation and handle validation cancellation correctly. See [#252](https://github.com/eclipse/lsp4mp/pull/252).
+ * enhancement - Property file with property expressions (without default value) are flagged as wrong. See [#227](https://github.com/eclipse/lsp4mp/issues/227).
+
```

Comment:
PR 225/Issue 224

Reply:
I'll include it as part of the entry that 227 is on as they seemed closely related.